invenio_upgrader/upgrades/invenio_2015_07_14_innodb.py

```diff
@@ -1,65 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
was `invenio_upgrader_2015_09_23_legacy_removal.py` released in any 0.x? If yes, you need to add new one (better create new one anyway).